frontend/app/_components/wallet-connection.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+
+import { useActiveWalletType, useDisconnect, useSuggestChainAndConnect, useAccount } from "graz"
+import { useCallback } from "react"
+import { appChain } from "../../utils/chains"
+import { Button } from "../../components/button"
+import { Card, CardContent, CardHeader, CardTitle } from "../../components/card"
+
+export const WalletConnection: React.FC = () => {
```

Comment:
This should not be an eaxmple, rather the wallet connection button should be in thee header.